src/bindings/python/requirements_test.txt

```diff
@@ -1,4 +1,6 @@
 -c ./constraints.txt
+protobuf
+onnx
```

Comment:
why do we need it?
I supposed all ONNX related tests are moved to a different location

Reply:
Tests_compatibility were not moved since they are in deprecation state, and they are still testing ONNX using API 1.0

Reply:
> they are still testing ONNX using API 1.0

can we remove it? will we drop some coverage?

Reply:
I think this requires the expertise of ONNX FE team. @gkrivor could you please take a look?

Can we merge this PR to fix the postcommit and resolve the conversation offline?

Reply:
@p-wysocki, please create JIRA ticket and assign to @gkrivor to proceed.
We need investigation if we are ready to drop these tests.

Reply:
Continued in 116953.